src/site/antora/modules/ROOT/pages/manual/layouts.adoc

```diff
@@ -557,6 +557,11 @@ include::partial$manual/plugin-preliminaries.adoc[]
 
 Layouts are xref:manual/plugins.adoc[plugins] implementing link:../javadoc/log4j-core/org/apache/logging/log4j/core/Layout.html[the `Layout` interface].
 If your layout is a `String`-based one, we recommend you to extend your plugin class from link:../javadoc/log4j-core/org/apache/logging/log4j/core/layout/AbstractStringLayout.html[`AbstractStringLayout`], which contains convenience for some of the boilerplate code shared by `String`-based layouts.
+While annotating your layout with `@Plugin`, you need to make sure that
+
+* It has a unique `name` attribute across all available `Layout` plugins
```

Comment:
Technically, it'd have to be a unique `name` attribute for all `Core` category/namespace plugins as this name determines the XML element name it aliases to. Had we used different categories for different plugin types, then that would likely correspond to XML namespaces which would make the config file more complicated to write.

Reply:
Clarified in the various plugin pages.